database/sql/src/main/scala/cromwell/database/slick/MetadataSlickDatabase.scala

```diff
@@ -100,17 +100,21 @@ trait MetadataSlickDatabase extends MetadataSqlDatabase {
     }
   }
 
-  def refreshMetadataSummaries(startId: Long, startTimestamp: Option[Timestamp],
+  def refreshMetadataSummaries(startTimestamp: Option[Timestamp],
                                key1: String, key2: String, key3: String, key4: String,
                                buildUpdatedSummary: (Option[WorkflowMetadataSummary], Seq[Metadatum]) =>
                                  WorkflowMetadataSummary)
-                              (implicit ec: ExecutionContext): Future[Seq[Metadatum]] = {
+                              (implicit ec: ExecutionContext): Future[Long] = {
     val action = for {
+      startIdOption <- getSummaryStatusMaximumId("WORKFLOW_METADATA_SUMMARY", "METADATA_JOURNAL")
```

Comment:
I've decided to merge as is. I looked at how one might retrieve the table name from slick, and the direction I [found](http://stackoverflow.com/a/28706363/3320205) went through the `TableQuery.baseTableRow`-- except the inline docs on that method [explicitly](https://github.com/slick/slick/blob/3.1.1/slick/src/main/scala/slick/lifted/Query.scala#L298-L299) hint that it's not for public consumption.

Instead, I _feel_ like the table names (and column names, and index names, etc.) should be defined not by slick, but up in the base `cromwell.database.sql.tables` layer. However I also feel like the slick code adds so much verbosity already, that I hesitate to add yet-another set of boilerplate/wiring to be filled in to get a table up and running, in addition to the liquibase modeling too.

I'm very ok with someone smoothing this out in a future PR.